cmd/esc/cli/cli_test.go

```diff
@@ -253,13 +253,14 @@ type testEnvironmentRetract struct {
 type testEnvironmentRevision struct {
 	number    int
 	yaml      []byte
-	tag       string
+	tags      map[string]bool
```

Comment:
curious why this changed? this is tracking the revision's etag, which should be static?

Reply:
Oh interesting - this was used by the test files as a way to pass in a revision tag so I modified it to take multiple tags to mirror production

Reply:
> I did notice it was also being used as the etag but I figured that was solely out of convenience

ah interesting--this double duty was unintentional, and this should be the etag. for convenience you can leave these changes if you like and I can straighten things out later